src/ngAnimate/module.js

```diff
@@ -267,9 +267,14 @@
  * .message.ng-enter-prepare {
  *   opacity: 0;
  * }
- *
  * ```
  *
+ * ### Animating any scope's variable change
+ *
+ * Unless possible with more proper ["animation aware" directive](#directive-support), that specific use case can always be covered with {@link ngAnimate.directive:ngAnimateSwap} as can be seen in {@link ngAnimate.directive:ngAnimateSwap#examples this example}.
```

Comment:
I would expand here on what type of usecase `ngAnimateSwap` helps support. I.e. that you can animate between multiple (>2) different states (variable values), whose values don't need to be known or referenced in the CSS styles.

BTW, it is better for readability and "review-ability" if lines are wrapped at 100 chars.